website/docs/docs/use-dbt-semantic-layer/tableau.md

```diff
@@ -12,30 +12,33 @@ The Tableau integration with the dbt Semantic Layer is a [beta feature](/docs/db
 
 The Tableau integration allows you to use worksheets to query the Semantic Layer directly and produce your dashboards with trusted data.  
 
-This integration provides a live connection to the dbt Semantic Layer through Tableau Desktop.
+This integration provides a live connection to the dbt Semantic Layer through Tableau Desktop or Tableau Server. 
 
 ## Prerequisites
 
 - You have [configured the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl) and are using dbt v1.6 or higher.
-- You must have [Tableau Desktop](https://www.tableau.com/en-gb/products/desktop) installed with version 2021.1 or greater
-   - Note that Tableau Online does not currently support custom connectors natively.
-- Log in to Tableau Desktop using either your license or the login details you use for Tableau Server or Tableau Online.
+- You must have [Tableau Desktop](https://www.tableau.com/en-gb/products/desktop) version 2021.1 and greater or Tableau Server.
+  - Note that Tableau Online does not currently support custom connectors natively. If you use Tableau Online, you will only be able to access the connector in Tableau Desktop.
+- Log in to Tableau Desktop (with Online or Server credentials) or a license to Tableau Server
 - You need your dbt Cloud host, [Environment ID](/docs/use-dbt-semantic-layer/setup-sl#set-up-dbt-semantic-layer) and [service token](/docs/dbt-cloud-apis/service-tokens) to log in. This account should be set up with the dbt Semantic Layer.
 - You must have a dbt Cloud Team or Enterprise [account](https://www.getdbt.com/pricing) and multi-tenant [deployment](/docs/cloud/about-cloud/regions-ip-addresses). (Single-Tenant coming soon)
 
 
-## Installing
+## Installing the Connector
 
 1. Download the GitHub [connector file](https://github.com/dbt-labs/semantic-layer-tableau-connector/releases/download/v1.0.2/dbt_semantic_layer.taco) locally and add it to your default folder:
-   - Windows: `C:\Users\\[Windows User]\Documents\My Tableau Repository\Connectors`
-   - Mac: `/Users/[user]/Documents/My Tableau Repository/Connectors`
-   - Linux: `/opt/tableau/connectors`
+
+|OS|Tableau Desktop|Tableau Server|
+|Windows|`C:\Users\\[Windows User]\Documents\My Tableau Repository\Connectors`|`C:\Program Files\Tableau\Connectors`|
+|Mac|`/Users/[user]/Documents/My Tableau Repository/Connectors`|n/a|
+|Linux|`/opt/tableau/connectors`| `/opt/tableau/connectors`|
+ 
 2. Install the [JDBC driver](/docs/dbt-cloud-apis/sl-jdbc) to the folder based on your operating system:
```

Comment:
@aiguofer how does the driver installation instructions change with Server? Btw, I assume that you don't HAVE to publish from Desktop, you can just use the connector directly in Server if you install it properly.

Reply:
I don't think it changes... https://help.tableau.com/current/pro/desktop/en-us/examples_otherdatabases_jdbc.htm#specify-the-right-jdbc-driver

Reply:
Did you have to do something different when you added it to your Tableau Server instance? I might be confused on the steps required to use our connector in Server

Reply:
nope, steps are same as desktop:
- Put connector in right place
- Put driver in right place

If you do this while tableau is "open" (in the case of server, "running"), then you must restart it.

Reply:
Thanks! I might add that for thoroughness